Client/Frontend/Browser/BrowserViewController/BrowserViewController+CookieNotificationBlocking.swift

```diff
@@ -0,0 +1,51 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import SwiftUI
+import BraveUI
+import BraveShared
+import Data
+
+extension BrowserViewController {
+  func presentCookieNotificationBlockingCalloutIfNeeded() {
+    // Don't show this if we already enabled the setting
+    guard !FilterListResourceDownloader.shared.isEnabled(filterListUUID: FilterListResourceDownloader.cookieConsentNoticesUUID) else { return }
+    // Don't show this if we are presenting another popup already
+    guard !isOnboardingOrFullScreenCalloutPresented else { return }
```

Comment:
Can you add the check for onboarding status here ?
`let basicOnboardingNotCompleted = Preferences.General.basicOnboardingProgress.value != OnboardingProgress.newTabPage.rawValue`

The reason is If the user kills the app in the middle of onboarding and open again. The behaviour should be showing NTP url calllouts but this cookie consent will take precedence there.